examples/sleepy-demo/app.c

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
+ * @file
+ * @brief Core application logic.
+ *******************************************************************************
+ * # License
+ * <b>Copyright 2021 Silicon Laboratories Inc. www.silabs.com</b>
+ *******************************************************************************
+ *
+ * The licensor of this software is Silicon Laboratories Inc. Your use of this
```

Comment:
We typically keep code that does not carry the OpenThread license in `third_party` directory. Is it possible to do one of the following?

1. Update this license header to OpenThread license header
2. Move these source files to `third_party`?

Reply:
Thanks for catching that @jwhui !

I went with option 1